js/app/modules/card/knowledgeDocument.js

```diff
@@ -237,7 +247,7 @@ const KnowledgeDocument = new Module.Class({
     _get_webview: function () {
         let webview = this._create_webview();
 
-        webview.renderer.enable_scroll_manager = this.show_toc;
+        webview.renderer.enable_scroll_manager = true;
```

Comment:
You are right, but doesn't work out of the box.

1. we still need the `.startWith` fix.
2. even if we return false, the EknWebView custom handler for 'ekn:' scheme must consider this case, or else this case falls into a [case](https://github.com/endlessm/eos-knowledge-lib/blob/master/js/app/widgets/eknWebview.js#L154) which I honestly can't understand what it is for, any idea?

Anyway, I did a dirty/quick hack just to check it works (and it does), but I need to make sure I can insole from that case I mentioned inside that handler so I don't break anything.

The only doubt I have re. this approach (via webkit) is that clicking on that link will teleport the user to the section, differently from the animated way of doing when clicking on the TOC, so te user will get different behaviors depending on where it clicks. 

if we try to unify these two behaviors we basically end up with same PR we have now.

Let me know what you think!